plugins/bazaar/src/components/ProjectCard/ProjectCard.tsx

```diff
@@ -37,39 +38,48 @@ type Props = {
   height?: string;
 };
 
-const useStyles = makeStyles({
-  statusTag: {
-    display: 'inline-block',
-    whiteSpace: 'nowrap',
-    marginBottom: '0.8rem',
-  },
-  description: {
+type StyleProps = {
+  height?: string;
+};
+
+const useStyles = makeStyles((theme: BackstageTheme) => ({
+  description: (props: StyleProps) => ({
+    height: props.height === '7rem' ? '3.7rem' : '10.2rem',
     display: '-webkit-box',
     WebkitLineClamp: 7,
     WebkitBoxOrient: 'vertical',
     overflow: 'hidden',
+    '-webkit-background-clip': 'text',
```

Comment:
This doesn't look quite right x)

I think you'll want to use the above syntax anyway, with the `Webkit` prefix